lib/seattleflu/id3c/cli/command/etl/redcap_det_fh_airs.py

```diff
@@ -0,0 +1,861 @@
+"""
+Process DETs for the Fred Hutch AIRS study into the FHIR receiving table
+"""
+import logging
+import os
+
+from dateutil.relativedelta import relativedelta
+from enum import Enum
+from id3c.cli.command.etl import redcap_det
+from id3c.cli.redcap import is_complete, Record as REDCapRecord
+from id3c.db import find_identifier
+from seattleflu.id3c.cli.command import age_ceiling
+from .redcap import *
+from .redcap_map import *
+
+LOG = logging.getLogger(__name__)
+
+
+class AIRSProject():
+    id: int
+    lang: str
+    command_name: str
+
+    def __init__(self, project_id: int, lang: str, command_name: str) -> None:
+        self.id = project_id
+        self.lang = lang
+        self.command_name = command_name
+
+
+REDCAP_URL  = os.environ.get('FH_AIRS_REDCAP_URL') # REDCap server hostname
```

Comment:
Thanks for adding this. Small change but by convention, can the suffix be `_REDCAP_API_URL` to match the others we use? Thanks!